README.md

```diff
@@ -1,2 +1,48 @@
-# AwesomeBooks-Module
-Microverse project for practicing implementing modules (javascript)
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# AwesomeBooks-Project
+Awesome books: plain JavaScript with objects
+
+## Acces link to live version
+
+> https://vitorguedesmadeira.github.io/AwesomeBooks-Module/
+
+## Acces link for cloning repository
+
+https://github.com/VitorGuedesMadeira/AwesomeBooks-Module.git
+
```

Comment:
- [ ] Merely providing the link for cloning is not enough. Kindly include more SPECIFIC instructions on how to clone the repo locally, change into the directory and run the app. This is important for non-technical/unfamiliar users.

Reply:
@TedLivist Would you kindly check it again? I Asked student success about it and she told me I can talk to you since this should be optional and not a requirement for the second module, second week. Also, I used 2/3 code reviews for the same feedback.